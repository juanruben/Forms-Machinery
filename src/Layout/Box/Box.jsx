import React from 'react';
import './Box.scss';

const Box = (props) => {
    const { children } = props;
    return (
        <div className="box-container">
            {children}
        </div>
    );
};

export default Box;
