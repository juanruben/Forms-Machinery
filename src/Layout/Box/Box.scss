.box-container {
  background-color: white;
  border: solid rgb(231, 231, 231) 1px;
  border-radius: 4px;
  padding: 0 80px;
}
