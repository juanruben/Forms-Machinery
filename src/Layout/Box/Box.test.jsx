import React from 'react';
import { shallow } from 'enzyme';
import Box from './Box';

describe('<Box />', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<Box />);
    });

    it('should render correctly', () => expect(wrapper.debug()).toMatchSnapshot());

    it('should render divs', () => {
        expect(wrapper.find('div')).toHaveLength(1);
    });
});
