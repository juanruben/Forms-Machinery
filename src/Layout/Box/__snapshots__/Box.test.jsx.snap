// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Box /> should render correctly 1`] = `"<div className=\\"box-container\\" />"`;
